src/main/riddl/Members/members.riddl

```diff
@@ -12,15 +12,15 @@ context MembersContext  is {
       | The values of this enumeration define the ways in which a member may
       | be notified of changes.
     }
-    type Status is any of { Active, Suspended }
+    type Status is any of { Draft, Active, Inactive, Suspended, Terminated }
     type Info is {
         contact: Contact,
         handle: String briefly "username. Must be unique.",
         avatar: URL briefly "thumbnail image of the user. Need to screen to ensure community guidelines compliance. i.e. no porn or offensive content. Use Gravitar?",
         firstName: String briefly "Real first name of the member",
         lastName: String briefly "Real last name of the member",
-        mobileNumber: MobileNumber? briefly "Optional mobile phone number",
-        emailAddress: EmailAddress? briefly "Optional email address",
+        //mobileNumber: MobileNumber? briefly "Optional mobile phone number",
+        //emailAddress: EmailAddress? briefly "Optional email address",
```

Comment:
I am ok dropping email and mobile for the KalixStudy. Thanks for commenting so we don't lose the idea.